cranelift-codegen/src/ir/immediates.rs

```diff
@@ -738,6 +738,12 @@ impl From<f64> for Ieee64 {
     }
 }
 
+impl From<i32> for Ieee64 {
```

Comment:
I was confused by this over at https://github.com/CraneStation/cranelift/pull/833#discussion_r302852200. I thought this did `f64::from_bits`, not `as f64`.

Reply:
I'm changing this to use `f64::from_bits`